File: java/soma_vetores.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetores {

	public static void main(String[] args) {

		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);

		int N;

		System.out.print("Quantos valores vai ter cada vetor? ");
		N = sc.nextInt();

		int[] a = new int[N];
		int[] b = new int[N];
		int[] c = new int[N];

		System.out.println("Digite os valores do vetor A:");

		for (int i = 0; i < N; i++) {
			a[i] = sc.nextInt();
		}

		System.out.println("Digite os valores do vetor B:");

		for (int i = 0; i < N; i++) {
			b[i] = sc.nextInt();
		}

		for (int i = 0; i < N; i++) {
			c[i] = a[i] + b[i];
		}

		System.out.println("VETOR RESULTANTE:");

		for (int i = 0; i < N; i++) {
			System.out.printf("%d\n", c[i]);
		}

		sc.close();
	}
}

File: python/soma_vetores.py
N: int

N = int(input("Quantos valores vai ter cada vetor? "))

a: [int] = [0 for x in range(N)]
b: [int] = [0 for x in range(N)]
c: [int] = [0 for x in range(N)]

print("Digite os valores do vetor A:")

for i in range(N):
    a[i] = int(input())

print("Digite os valores do vetor B:")

for i in range(N):
    b[i] = int(input())

for i in range(N):
    c[i] = a[i] + b[i]

print("VETOR RESULTANTE:")

for i in range(N):
    print(c[i])
